docs/upgrade.html

```diff
@@ -70,6 +72,8 @@
     <li>The Zookeeper dependency was removed from the Streams API. The Streams API now uses the Kafka protocol to manage internal topics instead of
         modifying Zookeeper directly. This eliminates the need for privileges to access Zookeeper directly and "StreamsConfig.ZOOKEEPER_CONFIG"
         should not be set in the Streams app any more. If the Kafka cluster is secured, Streams apps must have the required security privileges to create new topics.</li>
+    <li>Upgrading your Streams application from 0.10.1 to 0.10.2 does not require a broker upgrade.
```

Comment:
You mean in "client compatibility" ? Not sure if people who are interested in Streams read this part. Furthermore, we need to point out in the second sentence, that Streams 0.10.2 does not work with 0.10.1 broker -- adding the first sentence make the whole bullet point self contained.